linode/nb/framework_resource_schema.go

```diff
@@ -106,6 +106,9 @@ var resourceNodebalancerV0 = schema.Schema{
 		"label": schema.StringAttribute{
 			Description: "The label of the Linode NodeBalancer.",
 			Optional:    true,
+			Validators: []validator.String{
+				stringvalidator.LengthBetween(3, 32),
+			},
```

Comment:
For resource like NodeBalancer, the label requirement is not specified in the doc, but it does have some limits, and Cloud Manager doesn't allow any label with special characters. Can we dig into the API's constrains and add a regex validator here?